fiftyone/core/dataset.py

```diff
@@ -3193,8 +3193,9 @@ def delete_view(self, name):
         view_doc = self._get_view_doc(name, pop=True)
         self._doc.save()
 
-    def delete_views(self):
+    def delete_saved_views(self):
         """Deletes all saved views from this dataset."""
+        print("Deleting saved views: ", self.list_saved_views())
```

Comment:
remove print

Reply:
oh I meant to ask, should we put some kind of warning? It seems too easy to accidentally delete them all.

Reply:
That could be a design choice we could make, but it's not used anywhere else right now so I wouldn't add it here:

```py
dataset.delete_samples(...)
dataset.delete_evaluations()
dataset.delete()
```